index.html

```diff
@@ -106,39 +113,45 @@ <h3>MY RECENT WORK</h3>
   <hr>
   <section class="card">
     <!--PLACE HOLDER IMAGE-->
-    <div class="place-holder-img">
-      <img src="./img/ImgPlaceholder.png" alt="place holder">
-    </div>
-    <!-- MUTLI POST STORIES -->
-    <div class="title-post">
-      <h2>Multi-Post Stories</h2>
-    </div>
-    <div class="paragraph2">
-      <p>A daily selection of privately
-        personalized reads; no accounts
-        or sign-ups required. has been the
-        industry's standard dummy text
-        ever since the 1500s, when an
-        unknown printer took a standard
-        dummy text.</p>
+    <div class="project-card">
+      <div class="place-holder-img">
+        <img src="./img/ImgPlaceholder.png" alt="place holder">
+      </div>
+      <!-- MUTLI POST STORIES -->
+      <div class="desktop-version-card-details">
+      <div class="title-post">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="paragraph2">
+        <p>A daily selection of privately
+          personalized reads; no accounts
+          or sign-ups required. has been the
+          industry's standard dummy text
+          ever since the 1500s, when an
+          unknown printer took a standard
+          dummy text.</p>
+      </div>
+      <!-- languajes de programacion -->
+      <ul class="programming-languages">
+        <li><a href="">CSS</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png" alt="rectangular"></a></li>
+        <li><a href="">html</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png" alt="rectangular"></a></li>
+        <li><a href="">bootstrap</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png" alt="rectangular"></a></li>
+        <li><a href="">Ruby</a></li>
+      </ul>
+      <!-- SEE PROJECT BUTTON -->
+      <button class="btn">See project</button>
+      </div>
     </div>
```

Comment:
- [ Optional ] For desktop version of your portfolio Kindly work on the project content for recent work section , Kindly adjust alignment of the project content just like the Figma design.

Your project Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/200418964-f371da2b-df8b-4c46-8f16-57ba0af40542.PNG)

The Figma design look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/200419157-75d99055-7c10-4708-8a91-e519e61cde31.PNG)